.CMake/alg_support.cmake

```diff
@@ -161,39 +161,147 @@ cmake_dependent_option(OQS_ENABLE_SIG_rainbow_Vc_cyclic_compressed "" ON "OQS_EN
 
 option(OQS_ENABLE_SIG_SPHINCS "" ON)
 cmake_dependent_option(OQS_ENABLE_SIG_sphincs_haraka_128f_robust "" ON "OQS_ENABLE_SIG_SPHINCS" OFF)
+if(ARCH STREQUAL "x86_64" AND CMAKE_SYSTEM_NAME MATCHES "" AND OQS_USE_AES_INSTRUCTIONS)
```

Comment:
The templating scripts will have to be updated to remove the check for `CMAKE_SYSTEM_NAME` where there are no names to check against. I'll take a look at this, and I might clean up `copy_from_pqclean.py` some more.

Reply:
I've pushed an attempt at fixing this.